synapse_pay_rest/tests/fixtures/client.py

```diff
@@ -1,8 +1,10 @@
 import os
 from synapse_pay_rest.client import Client
 
-CLIENT_ID = os.environ['TEST_CLIENT_ID']
-CLIENT_SECRET = os.environ['TEST_CLIENT_SECRET']
+# CLIENT_ID = os.environ['TEST_CLIENT_ID']
+# CLIENT_SECRET = os.environ['TEST_CLIENT_SECRET']
+CLIENT_ID = input("Please enter client ID")
```

Comment:
Cool idea. For maintainability it's a good idea to remove commented code or add a note for why it is being commented. In this case safe to remove.

Also, I would prefer a solution that only prompts for user input if the variable is not already stored in ENV. That way we can set it in ENV and not have to enter it every time.